subsys/bluetooth/services/ots/ots_client.c

```diff
@@ -20,45 +18,25 @@
 #include <bluetooth/gatt.h>
 #include <bluetooth/l2cap.h>
 
-#include "../host/conn_internal.h"  /* To avoid build errors on use of struct bt_conn" */
+#include "../../host/conn_internal.h"  /* To avoid build errors on use of struct bt_conn" */
 
-#include "ots.h"
-#include "ots_internal.h"
-#include "otc.h"
+#include <bluetooth/services/ots.h>
+#include <bluetooth/services/ots_client.h>
+#include "ots_client_internal.h"
+#include "ots_l2cap_internal.h"
+#include "ots_dir_list_internal.h"
+#include "ots_oacp_internal.h"
+#include "ots_olcp_internal.h"
 
-#define BT_DBG_ENABLED IS_ENABLED(CONFIG_BT_DEBUG_OTC)
+#define BT_DBG_ENABLED IS_ENABLED(CONFIG_BT_DEBUG_OTS_CLIENT)
 #define LOG_MODULE_NAME bt_otc
 #include "common/log.h"
 
 /* TODO: KConfig options */
-#define OTS_CLIENT_INST_COUNT   1
-
-NET_BUF_POOL_FIXED_DEFINE(ots_c_read_queue, 1,
-			  sizeof(struct bt_gatt_read_params), 8, NULL);
-
-#define OTS_WORK(_w) CONTAINER_OF(_w, struct otc_instance_t, work)
+#define OTS_CLIENT_INST_COUNT     1
 
-static int l2cap_accept(struct bt_conn *conn, struct bt_l2cap_chan **chan);
-static struct net_buf *l2cap_alloc_buf(struct bt_l2cap_chan *chan);
-static int l2cap_recv(struct bt_l2cap_chan *chan, struct net_buf *buf);
-static void l2cap_sent(struct bt_l2cap_chan *chan);
-static void l2cap_status(struct bt_l2cap_chan *chan, atomic_t *status);
-static void l2cap_connected(struct bt_l2cap_chan *chan);
-static void l2cap_disconnected(struct bt_l2cap_chan *chan);
-
-#define CREDITS         10
-#define DATA_MTU        (BT_OTC_MAX_WRITE_SIZE * CREDITS)
-
-NET_BUF_POOL_FIXED_DEFINE(otc_l2cap_pool, 1, DATA_MTU, 8, NULL);
-
-struct l2ch {
-	struct k_delayed_work recv_work;
-	struct bt_l2cap_le_chan ch;
-};
-
-#define L2CH_CHAN(_chan) CONTAINER_OF(_chan, struct l2ch, ch.chan)
-#define L2CH_WORK(_work) CONTAINER_OF(_work, struct l2ch, recv_work)
-#define L2CAP_CHAN(_chan) _chan->ch.chan
+#define BT_OTC_MAX_WRITE_SIZE    23
```

Comment:
Inconsistent naming scheme

Reply:
Whoa, the marked code exploded... What I meant was that `OTS_CLIENT_INST_COUNT` and `BT_OTC_MAX_WRITE_SIZE` does not follow the same naming scheme

Reply:
Now they do :-)